events/events.go

```diff
@@ -146,3 +157,57 @@ func reconcileResponderEvent(tx *gorm.DB, event api.Event) error {
 
 	return nil
 }
+
+func reconcileCommentEvent(tx *gorm.DB, event api.Event) error {
+	commentID := event.Properties["id"]
+	commentBody := event.Properties["comment"]
+
+	// Get all responders related to a comment
+	var responders []api.Responder
+	commentRespondersQuery := `
+        SELECT * FROM responders WHERE incident_id IN (
+            SELECT incident_id FROM comments WHERE id = ?
+        )
+    `
+	err := tx.Raw(commentRespondersQuery, commentID).Scan(&responders).Error
+	if err != nil {
+		return err
+	}
+
+	// For each responder add the comment
+	for _, responder := range responders {
+		// Reset externalID to avoid inserting previous iteration's ID
+		externalID := ""
+
+		switch responder.Properties["responderType"] {
+		case responderPkg.JiraResponder:
+			externalID, err = responderPkg.NotifyJiraResponderAddComment(responder, commentBody)
+		case responderPkg.MSPlannerResponder:
+			externalID, err = responderPkg.NotifyMSPlannerResponderAddComment(responder, commentBody)
+		default:
+			continue
+		}
+
+		if err != nil {
+			// For now, we are only logging the error and moving on
+			// In future, we would want to associate error messages with responderType
+			// and when reprocessing, handle it only for specific responders
+			logger.Errorf("Error adding comment to responder:%s %v", responder.Properties["responderType"], err)
+			continue
+		}
+
+		// Insert into comment_responders table
+		if externalID != "" {
+			insertQuery := `
```

Comment:
Why not just inline the query ?

Reply:
primarily for readability, keeping the query and the statement separate

Reply:
I think it makes it less readable, if it is a very long query that is one thing, but if the query can fit on single line, and inlined in the parameters its fewer lines to read

Reply:
Cool